desc/coils.py

```diff
@@ -311,7 +366,7 @@ def compute_magnetic_field(self, coords, params={}, basis="rpz"):
         field : ndarray, shape(n,3)
             magnetic field at specified points, in either rpz or xyz coordinates
         """
-        if isinstance(params, dict):
+        if isinstance(params, dict) or params is None:
```

Comment:
this is in the commit message, but to explain this, I have been using SumMagneticField  to sum both a Coil and a MagneticField, and this was needed because the params is an empty dict by default for a coil, but it is None for a MagneticField (iirc)

Reply:
This was a mistake on my part, I should have made it default to None for all of them.